Helper/Order.php

```diff
@@ -1639,6 +1640,7 @@ private function getBoltTransactionStatus($transactionState)
             self::TS_CANCELED => 'CANCELED',
             self::TS_REJECTED_REVERSIBLE => 'REVERSIBLE REJECTED',
             self::TS_REJECTED_IRREVERSIBLE => 'IRREVERSIBLE REJECTED',
+            self::TS_CREDIT_IN_PROGRESS => 'Refund is in progress. See actual refund status in Bolt merchant dashboard',
```

Comment:
I'm not familiar enough with these transaction statuses, but what is the reason for having this one formatted so differently from the rest? Or does this not matter too much?

Reply:
This function provides statutes only for order comments. Because this case is a tricky one, I decided to add more text. See in PR description how the comment can look.